packages/create-react-native-library/src/utils/generateExampleApp.ts

```diff
@@ -65,17 +65,22 @@ export default async function generateExampleApp({
   const directory = path.join(dest, 'example');
   const args =
     type === 'native'
-      ? // `npx react-native init <projectName> --directory example --skip-install`
+      ? // `npx --package react-native-test-app@latest init --name ${projectName}Example --destination example --version ${reactNativeVersion}`
         [
-          'react-native@latest',
+          '--package',
+          `react-native-test-app@latest`,
           'init',
+          '--name',
           `${projectName}Example`,
-          '--directory',
+          `--destination`,
           directory,
-          '--version',
-          reactNativeVersion,
-          '--skip-install',
-          '--npm',
+          ...(reactNativeVersion !== 'latest'
+            ? ['--version', reactNativeVersion]
+            : []),
```

Comment:
RNTA is failing when providing `latest` as value for `--version` option (it automatically fetches latest version).